api/envoy/config/core/v3/protocol.proto

```diff
@@ -55,6 +83,18 @@ message QuicProtocolOptions {
   // window size now, so it's also the minimum.
   google.protobuf.UInt32Value initial_connection_window_size = 3
       [(validate.rules).uint32 = {lte: 25165824 gte: 1}];
+
+  // QUIC keep-alive probing packets work differently from HTTP/2 keep-alive PINGs in a sense that the probing packet
+  // itself doesn't timeout waiting for a probing response but it would elicit the peer to
+  // send something, i.e. ACK or PATH_RESPONSE, to push back connection idle timeout. Quic has a shorter idle
+  // timeout than TCP, so it doesn't rely on such probing to discover dead connections. If the peer fails to respond, the connection will idle timeout eventually. Thus, they are configured differently from :ref:`connection_keepalive
+  // <envoy_v3_api_field_config.core.v3.Http2ProtocolOptions.connection_keepalive>`.
+  // There is no need of :ref:`connection_idle_interval <envoy_v3_api_field_config.core.v3.KeepaliveSettings.connection_idle_interval>`
+  // since a similar feature is built into QUICHE which uses :ref:`connection idle_timeout <envoy_v3_api_field_config.listener.v3.QuicProtocolOptions.idle_timeout>` and the current PTO of the connection to decide whether to probe before
```

Comment:
Idea for a future commit: instead of documenting what not to do, can we just do it for the user? Can we take the HCM idle timeout, and tell QUICHE to use that as the idle timeout?  cc @mattklein123 , the expert on the numerous types of timeouts we have on http connections, requests, upstream requests, etc.